src/test2.py

```diff
@@ -275,13 +279,16 @@ def evaluate_in_sage(query: str, format=False) -> str:
 
 
 def main():
-    command = create_query(sys.argv[1], sys.argv[2], sys.argv[3], sys.argv[4])
+    command = create_query(sys.argv[2], sys.argv[3], sys.argv[4], sys.argv[5])
     final_query = sage_functions + "\n" + command
-    output = evaluate_in_sage(final_query).replace("'", "")
-    output = output.replace(",", "")
-    output = output.replace("[", "").replace("]", "").strip()
-    output += " "
-    print(output)
+    if sys.argv[1] == 'tt': # trace enabled
```

Comment:
If the trace option is enabled, print the Sage command instead of evaluating it.